Manito/Manito/Screens/Interaction/OpenManittoViewController.swift

```diff
@@ -177,12 +179,12 @@ final class OpenManittoViewController: BaseViewController {
     private func requestRoomInfo(roomId: String) {
         Task {
             do {
-                let data = try await openManittoService.requestStartingRoomInfo(roomId: roomId)
+                let data = try await self.openManittoService.requestStartingRoomInfo(roomId: roomId)
```

Comment:
p2;

```suggestion
                let data = try await self.openManittoService.requestStartingRoomInfo(roomId: self.roomId)
```

요기도.. 하하하하